	The patch makes the target 64-bit GCC use /lib, not /lib64.
	2012-12-15 <douglas@ttylinux.org>

--- gcc-4.6.3-OLD/gcc/config/i386/linux64.h	2011-09-08 02:12:35.000000000 -0700
+++ gcc-4.6.3-NEW/gcc/config/i386/linux64.h	2013-05-22 18:44:15.490456214 -0700
@@ -63,7 +63,7 @@
    done.  */
 
 #define GLIBC_DYNAMIC_LINKER32 "/lib/ld-linux.so.2"
-#define GLIBC_DYNAMIC_LINKER64 "/lib64/ld-linux-x86-64.so.2"
+#define GLIBC_DYNAMIC_LINKER64 "/lib/ld-linux-x86-64.so.2"
 
 #if TARGET_64BIT_DEFAULT
 #define SPEC_32 "m32"
